types/Parcel/PredefinedPackage/DHLExpress.d.ts

```diff
@@ -0,0 +1,15 @@
+export declare type PredefinedPackageDHLExpress =
```

Comment:
Things like this are nice but very difficult to maintain as they are constantly being updated. The documentation on our site is sadly out of date in many places for this reason. If we did Typescript definitions, I'd maybe want to start with a more generic/simple approach and add onto it as time goes by if we desired (meaning, only focus on the internal interfaces of this library and leave all these options for predefined packages and service levels out for now.) Adding them would incur a large overhead for our team I'm not sure we're ready to commit to yet

Reply:
I would say as a consumer of easypost's APIs, I think it would be a very worthwhile investment for these service levels and package types to be available at an API endpoint. If you think it's difficult for your team to keep up with all of the updates, imagine what it's like for us consumers who are not in the loop _at all_ with what's being added or changed.

Reply:
Completely agree! We are building a solution to this problem as we speak to address the information disparity. I don't have an official ETA on this, though I'd hope it will be available in the next few months. As such, I don't think including these here is the best option. If we decided against that, we can always re-add these later.

Reply:
I would agree – if there's an API to get this data from, then let's leave these out!

Reply:
These have been removed in favor of adding them back later with a more scalable/maintainable approach.